inginious/frontend/templates/admin/admin_users.html

```diff
@@ -20,24 +20,100 @@
 <h2>{{ _("Users Administration") }}</h2>
 <div>
     <button type="button" data-type="single" data-toggle="modal"
-            data-target="#add_modal" class="btn btn-info add_user" title="Add"><i class="fa fa-user-plus"></i> Add user</button>
+            data-target="#add_modal" class="btn btn-info add_user" title="Add"><i class="fa fa-user-plus"></i> {{ _("Add user")}}</button>
 </div>
 {% if feedback %}
     <p>{{feedback}}</p>
 {% endif %}
+<div class="mt-2">
+    <form method="POST">
+        <div class="card mb-3 section">
+            <div id="query_header" class="card-header" onclick="dropdown(this)">
+                <span class="title"> {{ _("Query users") }} </span>
+                <span style="font-size: 1.5rem;" class="dropdown_button fa fa-caret-down close"> </span>
+            </div>
+
+            <div class="card-body content">
+                <form id="select_form" method="post">
+                    <div class="row">
+                        <div class="col-sm-12 form-group">
+                            <label class="col-xs-12">{{ _("Username") }}</label>
+                            <input name="username" class="form-control" type="text" id="username"
```

Comment:
This conflicts with other fields and makes 

https://github.com/UCL-INGI/INGInious/blob/b07bae8ee3d5c027cd487f0e55fe5718dccc81bb/inginious/frontend/templates/admin/admin_users.html#L291

fail, always looking for a username corresponding to the search field. Please update both HTML and JS, avoid identical ids, rather use class for similar data or embeds in a `data-` tag, so that the fetched value is not random.